.npmignore

```diff
@@ -0,0 +1,6 @@
+.travis.yml
+figures
+logo.png
+node_modules
+test.js
+*.md
```

Comment:
This file is unnecessary due to the `"files"` field in __package.json__, I believe.